cfme/tests/candu/test_utilization_metrics.py

```diff
@@ -302,3 +316,69 @@ def test_raw_metric_host_disk(metrics_collection, appliance, provider):
         if record.disk_usage_rate_average is not None:
             assert record.disk_usage_rate_average > 0, 'Zero Host Disk IO'
             break
+
+
+def query_metric_rollup_table(appliance, provider, metric, azone_name):
+    metrics_tbl = appliance.db.client['metric_rollups']
+    ems = appliance.db.client['ext_management_systems']
+
+    with appliance.db.client.transaction:
+        provs = (
+            appliance.db.client.session.query(metrics_tbl.id)
+            .join(ems, metrics_tbl.parent_ems_id == ems.id)
+            .filter(metrics_tbl.resource_name == azone_name,
+            ems.name == provider.name)
+        )
+    return appliance.db.client.session.query(metrics_tbl).filter(
+        metrics_tbl.id.in_(provs.subquery()))
+
+
+def generic_test_azone_rollup(appliance, provider, metric):
+    azone_name = provider.data["cap_and_util"]["capandu_azone"]
+    query = query_metric_rollup_table(appliance, provider, metric, azone_name)
+
+    for record in query:
+        if hasattr(record, metric):
+            assert record.metric > 0, (
+                'Zero Azone CPU usage/memory usage/Disk IO/Network IO'
+            )
+            break
+
+
+@pytest.mark.uncollectif(lambda provider:
+                         provider.one_of(RHEVMProvider, VMwareProvider, OpenStackProvider))
```

Comment:
These and the subsequent provider filters can be moved to the top of the file. Also regarding the `uncollectif`, I expect @mshriver will comment something similar here https://github.com/ManageIQ/integration_tests/pull/9506#discussion_r339072512

Reply:
pytest.mark.provider( [EC2Provider, AzureProvider], required_fields=[(['cap_and_util', 'capandu_vm'], 'cu-24x7')], 

> scope="module")``

Reply:
I have replaced the uncollectif statement in the tests.